scripts/vsc-venv-wrapper/vsc-venv.sh

```diff
@@ -0,0 +1,97 @@
+SCRIPT_NAME=${BASH_SOURCE[0]} # $0 cannot be used as it gives '-bash' when sourced
+
+usage() {
+  echo "Usage: source $SCRIPT_NAME {activate <requirements.txt> [modules.sh] | deactivate}"
+  echo ""
+  echo "Commands:"
+  echo "  activate <requirements_file> [modules_script] Activate the environment using the specified requirements file."
+  echo "                                                Optionally, load modules from the specified script before activating the environment."
+  echo ""
+  echo "  deactivate                                    Deactivate the virtual environment."
+  echo ""
+  echo "Example Usage:"
+  echo "  $ source $SCRIPT_NAME activate requirements.txt modules.sh"
+  echo "  $ python my_script.py"
+  echo "  $ source $SCRIPT_NAME deactivate"
+  return 1
+}
+
+echo_info() { echo -e "\e[32m[INFO] $1\e[0m"; }
+echo_warning() { echo -e "\e[33m[WARNING] $1\e[0m"; }
+echo_error() { echo -e "\e[31m[ERROR] $1\e[0m"; }
+
+
+# ============================ Main functions ============================
+
+activate() {
+  REQUIREMENTS_FILE="$1"
+  MODULES_SCRIPT="$2"
+
+  VENV_LOCATION=$(realpath -m "venvs/venv-${VSC_OS_LOCAL}-${VSC_ARCH_LOCAL}") # full path of venv
+
+  # === Step 0: Warn user if they have modules loaded === #
+
+  N_LOADED_MODULES=$(echo "$LOADEDMODULES" | tr ':' '\n' | wc -l)
+  if [ "$N_LOADED_MODULES" -gt 4 ]; then # 4 is the number of modules loaded by default
+    echo_warning "You have loaded modules in the current shell. If you want to use these modules, please provide a modules script as the second argument."
```

Comment:
shoudl clarify that these are modules other then the cluster modules

Reply:
I should also print the list of modules that are loaded but shouldn't be (so without `cluster/*` and `env/*`, those are fine)

Reply:
Fixed in [loaded modules warning calculated by list + print of extra modules](https://github.com/hpcugent/vsc_user_docs/pull/670/commits/5e0f1c6f31b0e995656449a0d684704b8d4b7010)